openapi/paths/usages.yaml

```
@@ -0,0 +1,68 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - Usage
+  summary: Retrieve usage records
+  operationId: GetUsagesCollection
+  x-sdk-operation-name: getAll
+  description: |
+    Retrieves a list of usage records.
+  parameters:
+    - $ref: ../components/parameters/collectionFilter.yaml
+    - $ref: ../components/parameters/collectionSort.yaml
+    - $ref: ../components/parameters/collectionLimit.yaml
+    - $ref: ../components/parameters/collectionOffset.yaml
+    - $ref: ../components/parameters/collectionQuery.yaml
+  responses:
+    '200':
+      description: List of usage records retrieved succesfully.
```

nitpick: 'successfully' does not add to the sentence
```suggestion
      description: List of usage records retrieved.
```